build.sbt

```diff
@@ -23,7 +24,7 @@ lazy val commonSettings = Seq(
   organization := "com.cognite",
   organizationName := "Cognite",
   organizationHomepage := Some(url("https://cognite.com")),
-  version := "2.2.2",
+  version := "2.2.3",
```

Comment:
This is a breaking change, so should bump version to 2.3.0

Reply:
updated to 2.3.0 :)